### ISSUE TITLE
Option to zoom with scroll wheel

### ISSUE THREAD

**Issue by khimaros** [excalidraw/excalidraw#5755]
coming from other creative tools like miro, cad, blender, etc. and even consumer products like maps, it's common to have scroll wheel zoom without needing to hold ctrl.

maybe changing this default now is more harm than good, but it would be nice to have an **option to change the mouse behavior** to this. i don't often need to scroll up/down on the canvas when i can middle-mouse drag.

**Comment by nicolasmarc1** [comment]
This would be really nice to have this! ctrl + scroll was made to differentiate scrolling from zooming in linear ui / documents where you navigate primary by scrolling up and down, it doesn't really make sense on a canvas where you can just use the scroll wheel to point and zoom (1 hand vs 2 before) or can move around with keyboard directional arrows (1 hand vs 2 before) or by holding the middle or right click while moving the mouse ( still 1 hand vs 2 before)

Miro / heptabase / Mural.. ) give the option to change between what they call trackpad navigation (zoom w ctrl + scroll wheel) and mouse navigation (point and zoom)

**Comment by dwelle** [comment]
Not strictly against making the scroll as the default, or adding navigation types.

Problem with making it the default is that many people don't know how to pan the canvas (they are either not used to `mousewheel-drag` or don't read the hints). So we'd at minimum need to wait until we introduce the panning tool.

**Comment by nicolasmarc1** [comment]
Of course! That's why the other comment and mine is more about giving the option of both modes (like the above examples in miro, heptabase, mural) instead of changing the default traditional navigation mode

**Comment by khimaros** [comment]
thank you for considering this! would you accept a PR adding this as a preference? if so, how would you prefer to see the preference exposed in the UI?

**Comment by pavelkostychev** [comment]
Scrolling to zoom is possible with holding control, but there is no option to box zoom at all – it's available in any tool I know. This is the only feature I'm missing!

**Comment by aidenaiwan** [comment]
Hello, what;s the status of this?

**Comment by dwelle** [comment]
shipped, forgot to close (https://github.com/excalidraw/excalidraw/pull/12099)

https://x.com/excalidraw/status/2100317419818356774
